web-wallet/src/lib/containers/MigrateContract/MigrateContract.svelte

```diff
@@ -36,7 +36,8 @@
   } from "$lib/migration/walletConnection";
   import { getBalanceOfCoin } from "$lib/migration/migration";
 
-  const { name: migrationNetwork } = $networkStore;
+  export let migrationNetwork = "mainnet";
```

Comment:
I don't think is wise here to give it an initial value: it's better if it's a required prop.